src/c++/perf_analyzer/genai-perf/genai_perf/main.py

```diff
@@ -84,13 +84,16 @@ def generate_inputs(args: Namespace, tokenizer: Tokenizer) -> None:
 
 
 def calculate_metrics(args: Namespace, tokenizer: Tokenizer) -> ProfileDataParser:
+    if args.endpoint_type == "embeddings":
+        return ProfileDataParser(args.profile_export_file)
+
```

Comment:
i would make this an if else. This is an exclusive choice right?

Reply:
Sure we can make it if-else